VirtoCommerce.CatalogModule.Web/Scripts/blades/property-detail.js

```diff
@@ -71,6 +71,7 @@
             }
 
             blade.currentEntity = angular.copy(data);
+            blade.currentEntity.isValid = true;
```

Comment:
This line makes currentEntity always dirty, therefore save button always enabled even if property unchanged.
use '$" as leading char for property names if you want to do not use them for equality comparison. 
`blade.currentEntity.$isValid` as example